package.json

```diff
@@ -461,6 +465,10 @@
           "command": "one.toolchain.runCfg",
           "when": "false"
         },
+        {
+          "command": "one.toolchain.inference",
```

Comment:
```suggestion
          "command": "one.toolchain.runInference",
```
Maybe?

Reply:
oh, I will fix it later. There are many places to modify together.